app/Notifications/RequestWithdrawalApprovedNotification.php

```
@@ -0,0 +1,73 @@
+use App\Models\RequestWithdrawal;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class RequestWithdrawalApprovalNotification extends Notification
+{
+    use Queueable;
+
+    private $token;
+    private $model;
+    public $id;
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)